### PR TITLE
Fix banner close button hover state

### LINKED ISSUE #2203: Close button not visible in dark banner
## Description

In staging homepage, the dark banner announcing the analytics and privacy settings shows a dark close button. It blends with the background and therefore not visible to users.

## Reproduction

I think you need to delete your cookies beforehand. Then go to [staging homepage](https://staging.openverse.org/) and see the banner's top-right.

## Screenshots

Homepage screenshot showing the privacy banner

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #2203 by @fcoveram 

## Description

This PR updates the `VCloseButton`'s "black" variant to use transparent background, white icon, and white background with opacity-10 on hover. This button is used in the pages menu modal on the mobile screen and the black notification banner on the yellow home page.

The banner close button's size changed from "tiny" to "small" as in the mockups.

## Testing Instructions

Open the app in an incognito mode to make sure that the privacy banner shows up. Check that the close button is visible, and it changes background color on hover.
Narrow the screen and click on the "menu" button to open the pages menu modal. The close button should look the same as the banner button.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/components/VCloseButton.vue [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default defineComponent({
      * The size of the underlying VIconButton.
      */
     size: {
-      type: String as PropType<"tiny" | "small" | "medium" | "large">,
+      type: String as PropType<"close" | "small" | "medium" | "large">,
```
Comment (dhruvkb) [inline frontend/src/components/VCloseButton.vue:65]: Can we bikeshed on the name?

**Review by dhruvkb — APPROVED** [review]
LGTM, except that "close" is a rather confusing name for the h-8, w-8 size but I don't want to block a deployment-blocking fix over bikeshedding a name.

File: frontend/src/components/VCloseButton.vue (hunk 1 above)
Comment (obulat) [reply frontend/src/components/VCloseButton.vue:65]: I would love to! Can you propose a better name? What can come between "tiny" and "small"?

**Comment by obulat** [comment]
@fcoveram , I've updated the paddings to match the i2 from Figma file: https://www.figma.com/file/nxkoGOzG5vnTpZgYpi4LsQ/Alert-banner?type=design&node-id=933%3A7003&t=pFK8ogpSQ1t5eBuX-1 (`p-2` for banners on below `lg` and `p-3` for banners `lg` and above)

**Pull request merged**